### PR TITLE
doc: Update mainnet docs for client release 1.1.0

### PR DESCRIPTION
**Pull request by YazzyYaz**

### CODE REVIEW

File: packages/docs/getting-started/running-a-validator-in-mainnet.md [hunk 1]
```diff
@@ -191,21 +191,15 @@ The `us.gcr.io/celo-org/celo-node:mainnet` image contains the [genesis block](ht
...
-Next, we'll run a node on your local machine so that we can use these accounts to lock CELO and authorize the keys needed to run your validator. To do this, we need to run the following commands, which fetch the genesis block and a list of other nodes in the network to connect to.
-
-```bash
-# On your local machine
-docker run -v $PWD:/root/.celo --rm -it $CELO_IMAGE init /celo/genesis.json
-export BOOTNODE_ENODES=`docker run --rm --entrypoint cat $CELO_IMAGE /celo/bootnodes`
-```
+Next, we'll run a node on your local machine so that we can use these accounts to lock CELO and authorize the keys needed to run your validator. To do this, we need to run the following command to run the node.
 
 To run the node:
```
Comment (jcortejoso) [inline packages/docs/getting-started/running-a-validator-in-mainnet.md]: Maybe this line sounds a bit repetitive based on the last line.

**Review by jcortejoso — APPROVED** [review]
lgtm

File: packages/docs/getting-started/running-a-validator-in-mainnet.md (hunk 1 above)
Comment (YazzyYaz) [reply packages/docs/getting-started/running-a-validator-in-mainnet.md]: Thanks, I removed it.

File: packages/docs/getting-started/running-a-validator-in-mainnet.md [hunk 2]
```diff
@@ -239,20 +229,20 @@ Notice the public address returned by this command, that can be exported and use
...
+The Celo validator community will contribute full node peers to help kickstart your nodes' peer discovery and block synchronisation processes. Here is the current list, set to `COMMUNITY_ENODES`:
 ```bash
 export COMMUNITY_ENODES="enode://f65013f1ac6827e275c2d2737ce13357f620d4364124d02227a19321c57f8fbf9214a9411de49d49f180b085b031d9d23211a6ead4499fc5f9d3592b55322123@50.17.60.161:30303"
 ```
 
-To use the nodes, set them to the value of the `bootnodes` or `bootnodesv4` options when running your node-starting `docker run...` commands (in this example, we will use `bootnodesv4` to keep the original command intact). To bootstrap your proxy node, you can run the command below (the same command at line 239 appended with `--bootnodesv4 $COMMUNITY_ENODES`):
+To use the nodes, set them to the value of the `--bootnodes` or `bootnodesv4` options when running your node-starting `docker run...` commands (in this example, we will use `bootnodesv4` to keep the original command intact). To bootstrap your proxy node, you can run the command below (the same command at line 239 appended with `--bootnodesv4 $COMMUNITY_ENODES`):
```
Comment (mcortesi) [inline packages/docs/getting-started/running-a-validator-in-mainnet.md]: typo, no need ot use `--`

**Review by mcortesi — APPROVED** [review]

**Pull request merged**